src/firestore/firekit.ts

```diff
@@ -104,6 +104,21 @@ interface ICreateUserInput {
   group: { id: string; abbreviation?: string } | null;
 }
 
+interface ICreateParentInput {
+  name: {
+    first: string;
+    last: string;
+  };
+}
+
+export interface IChildData {
+  email: string;
+  password: string;
+  userData: ICreateUserInput;
+  familyId: string;
+  orgCode: string;
+}
+
```

Comment:
I know we haven't started reformatting interface names in the entire codebase, but could we start with this PR by removing the leading `I`?

```suggestion
interface CreateParentInput {
  name: {
    first: string;
    last: string;
  };
}

export interface ChildData {
  email: string;
  password: string;
  userData: ICreateUserInput;
  familyId: string;
  orgCode: string;
}

```